File: sem 5/PHP/LAB EXCERCISE 3/Qu1.php
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title></title>
</head>
<body>
	<?php
		if(isset($_POST['num1']) && isset($_POST['num2']))
		{
			$num1=$_POST['num1'];
			$num2=$_POST['num2'];
			$sum=$num1+$num2;
		}
	?>
	<form method="post">
		<table border="1">
			<tr>
				<td>Enter the First Number:</td>
				<td><input type="number" name="num1" required>
			</tr>
			<tr>
				<td>Enter the Second Number:</td>
				<td><input type="number" name="num2" required>
			</tr>
			<tr>
				<td colspan="3" style="text-align: center;"><input type="submit" value="Add"></td>
			</tr>
			<tr>
				<td style="text-align: center;">Sum</td>
				<td>
					 <input type="number" value="<?php echo $sum; ?>" readonly style="text-align: center;">
				</td>
			</tr>
		</table>
	</form>
</body>
</html>

File: sem 5/Python/LAB TASK UNIT-4/Qu1.py
class Paper:
	def __init__(self,width,height):
		self.width=width
		self.height=height

def area(paper):
	return paper.width*paper.height

def perimeter(paper):
	return 2*(paper.width+paper.height)

num1=int(input("Enter the num1 : "))
num2=int(input("Enter the num2 : "))

obj=Paper(num1,num2)

print("Area : ",area(obj))
print("Perimeter : ",perimeter(obj))